components/eamxx/src/mct_coupling/atm_comp_mct.F90

```diff
@@ -356,4 +496,175 @@ subroutine atm_domain_mct( lsize, gsMap_atm, dom_atm )
     call mct_gGrid_importRAttr(dom_atm,"aream",data1,lsize) 
   end subroutine atm_domain_mct
 
+#ifdef HAVE_MOAB
+  subroutine  moab_atm_phys_scream()
```

Comment:
Can you add some comments at the top of this function, explaining what it does and when it's called?